addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -847,6 +847,7 @@ export class PosStore extends Reactive {
                     ]),
                     combo_item_id: comboItem.combo_item_id,
                     price_unit: comboItem.price_unit,
+                    price_type: "original",
```

Comment:
This is not really correct, the original `price_type` means that the price has not been recalculated other than with the methods in `pos_order_line.js`

As you can see in the master, the `price_type` is correctly set to manual at this point.

Unfortunately, we need to make an explicit choice for the calculation of combo_lines. We can keep their original prices to calculate the proportions in the combo and then use the price with pricelist of the parent combo to recalculate the line prices.

This is already done in the `computeComboItems` method, so perhaps you can reuse it when you `setPricelist`.